appservice/src/deploy/deployRunFromZip.ts

```diff
@@ -0,0 +1,165 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// import * as fs from 'fs';
+import { StorageAccount, StorageAccountListKeysResult } from 'azure-arm-storage/lib/models';
+import { StorageAccountListResult } from 'azure-arm-storage/lib/models';
+import { StringDictionary } from 'azure-arm-website/lib/models';
+import * as azureStorage from "azure-storage";
+import * as vscode from 'vscode';
+import { TelemetryProperties } from 'vscode-azureextensionui';
+import * as FileUtilities from '../FileUtilities';
+import { localize } from '../localize';
+import { SiteClient } from '../SiteClient';
+import { uiUtils } from '../utils/uiUtils';
+import { IQuickPickItemWithData } from '../wizard/IQuickPickItemWithData';
+import { formatDeployLog } from './formatDeployLog';
+
+export async function deployRunFromZip(client: SiteClient, fsPath: string, outputChannel: vscode.OutputChannel, telemetryProperties?: TelemetryProperties): Promise<void> {
+    // if (confirmDeployment) {
```

Comment:
Remove the commented code. Your current behavior is "Create a new blob every time" which is not destructive. We should also support "Replace the blob every time", but I'm okay addressing that separately